### ISSUE TITLE
New Account Wizard broken in TB 68

### ISSUE THREAD

**Issue by jdgeenen** [jdgeenen/gcontactsync#135]
**Describe the bug**
New Account Wizard broken in TB 68.  Events don't work in the wizard/wizardpage XUL.

**To Reproduce**
Steps to reproduce the behavior (TB68 and later)
1. Go to gContactSync -> New Account
2. Pick or type an account
3. Click Next
4. Browser page is not initialized.

**Expected behavior**
OAuth2 page from Google.

**Application (please complete the following information):**
 - Application: Thunderbird
 - Application Version: 68+
 - gContactSync Version: 3.1.15

**Comment by jdgeenen** [comment]
https://developer.thunderbird.net/add-ons/tb68/changes#less-than-dialog-greater-than-events should fix the wizard.

**Comment by jdgeenen** [comment]
After switching events I see this:
```NS_ERROR_FAILURE: Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIWebNavigation.loadURI] browser-custom-element.js:825
    loadURI chrome://global/content/elements/browser-custom-element.js:825
    _wrapURIChangeCall chrome://global/content/elements/browser-custom-element.js:764
    loadURI chrome://global/content/elements/browser-custom-element.js:824
    AccountSetupWizard_loadOAuthPage chrome://gcontactsync/content/AccountSetupWizard.js:202
    gCS_onPageShow chrome://gcontactsync/content/AccountSetupWizard.js:64```

**Comment by jdgeenen** [comment]
setAttribute("src", url); worked instead of loadURI.  I don't know why.

**Comment by jdgeenen** [comment]
The wizard is working in TB 68, but failing on TB built from source as of yesterday (with no errors to provide any hints).

**Comment by jdgeenen** [comment]
This should be fixed in 3.1.6.  I'll leave TB69 for another issue.
